eng/docker-compose/.env.example

```diff
@@ -54,15 +75,33 @@ BREAK_DURATION_SECONDS=30
 
 OPENSEARCH_URL=http://dms-search:9200
 
-# Variables for keycloak.yml
-KEYCLOAK_ADMIN=admin
-KEYCLOAK_ADMIN_PASSWORD=admin
-KEYCLOAK_PORT=8045
-
 # Authorization parameters
 IDENTITY_ENFORCE_AUTHORIZATION=true
-IDENTITY_SERVICE_ROLE=dms-client
-IDENTITY_AUTHORITY=http://dms-keycloak:8080/realms/dms
+# Config Service is currently assigning DMS clients the same role as Config Service clients, not using "dms-client" as intended
+# IDENTITY_SERVICE_ROLE=dms-client
+IDENTITY_SERVICE_ROLE=config-service-app
+IDENTITY_AUTHORITY=http://dms-keycloak:8080/realms/edfi
 IDENTITY_AUDIENCE=account
 IDENTITY_REQUIRE_HTTPS_METADATA=false
-IDENTITY_ROLE_CLAIM_TYPE=http://schemas.microsoft.com/ws/2008/06/identity/claims/role
+IDENTITY_ROLE_CLAIM_TYPE=http://schemas\\.microsoft\\.com/ws/2008/06/identity/claims/role
+
+# --------------
+# Config Service
+# --------------
+
+DMS_CONFIG_ASPNETCORE_HTTP_PORTS=8081
+DMS_CONFIG_DATASTORE=postgresql
+DMS_CONFIG_DATABASE_CONNECTION_STRING=host=dms-postgresql;port=5432;username=postgres;password=${POSTGRES_PASSWORD};database=${POSTGRES_DB_NAME};
+DMS_CONFIG_IDENTITY_ALLOW_REGISTRATION=true
+DMS_CONFIG_IDENTITY_SERVICE_ROLE=config-service-app
+DMS_CONFIG_IDENTITY_AUTHORITY=${IDENTITY_AUTHORITY}
+DMS_CONFIG_IDENTITY_AUDIENCE=${IDENTITY_AUDIENCE}
+DMS_CONFIG_IDENTITY_IDENTITY_SERVER=http://dms-keycloak:8080
+DMS_CONFIG_IDENTITY_REALM=edfi
+DMS_CONFIG_IDENTITY_CLIENT_ID=DmsConfigurationService
+DMS_CONFIG_IDENTITY_CLIENT_SECRET=s3creT@09
+DMS_CONFIG_IDENTITY_REQUIRE_HTTPS=false
+DMS_CONFIG_IDENTITY_ROLE_CLAIM_TYPE=${IDENTITY_ROLE_CLAIM_TYPE}
+DMS_CONFIG_IDENTITY_SCOPE=scp:edfi_dms_configuration_service/full_access
+DMS_CONFIG_LOG_LEVEL=Information
+DMS_CONFIG_DEPLOY_DATABASE=true
```

Comment:
@bradbanister this is a copy of the `.env` file that I used for my testing today.